spec/features/user_can_see_a_list_of_all_snacks_with_avg_price_spec.rb

```diff
@@ -0,0 +1,31 @@
+require 'rails_helper'
+
+RSpec.describe "As a User" do
+  describe "when I visit a vending machine show page" do
+
+    before :each do
+      @owner = Owner.create!(name: "Sam's Snacks")
+      @dons  = @owner.machines.create!(location: "Don's Sweet Delights")
+      @delight = @dons.snacks.create!(name: "Dandy Don's Delight", price: 4.00)
+      @chocoblaster = @dons.snacks.create!(name: "Chocoblaster", price: 2.00)
+    end
+
+    it "can see a list of all snacks in a vending machine" do
+
+      visit machine_path(@dons)
+
+      expect(page).to have_content(@delight.name)
+      expect(page).to have_content(@delight.price)
+      expect(page).to have_content(@chocoblaster.name)
+      expect(page).to have_content(@chocoblaster.price)
```

Comment:
For testing best practices, I would expect that your page has something like "Chocoblaster: $2.00" Since this is the wqy that you set up your view to display the snack info. It's okay to hard code your expectations, especially when testing expected content on a page.